### PR TITLE
Add missing HgCD lines

### PR DESCRIPTION
**Pull request by wtgee**
~~This is a naive combination of the `Hg.txt` file and the `HgCd.txt` file. I've kept the relative intensities from the original `HgCd.txt` file where appropriate.~~

~~I've read through the past tickets and there is concern these lines wouldn't be accurate for lamps with two elements, such as `HgCd`, and this does not address that. See, e.g. https://pfspipe.ipmu.jp/jira/browse/PIPE2D-935~~

Edit:
This line list is a combination of Hg.txt and HgCd.tx. It has been reviewed by @astrockragh,
who removed blends without well defined wavelengths. The intensities are both from NIST and
from the new_HgCd_liinemeas file by E Kado-Fong.

### CODE REVIEW

**Comment by wtgee** [comment]
@PaulPrice could you look this over when you get the chance?

File: pfs/lineLists/HgCd.txt [hunk 1]
```diff
@@ -15,18 +15,341 @@
...
+# BLEND=2: Line is blended with other line(s)
+# SUSPECT=4: Line is of suspect quality
+# REJECTED=8: Line has been rejected from use in PFS
+# BROAD=16: Line is broader than normal
+#
+# Source:
+#
+# Wavelengths are taken from NIST and at vacuum
+# Relative intensities are mostly taken from NIST, but some are taken from
+# new_HgCd_linemeas.csv by E Kado-Fong
```
Comment (PaulPrice) [inline pfs/lineLists/HgCd.txt:39]: Repeated lines.

**Review by PaulPrice — APPROVED** [review]
Looks reasonable. Could you please fix the summary line of the commit?

File: pfs/lineLists/HgCd.txt (hunk 1 above)
Comment (wtgee) [reply pfs/lineLists/HgCd.txt]: Thanks, copy/paste error. I removed these.

File: pfs/lineLists/HgCd.txt [hunk 2]
```diff
@@ -15,18 +15,321 @@
...
+ 802.01000            1.00        HgI       0
+ 802.45000            6.00        HgI       0
+ 803.41000             nan        HgI       0
+ 805.22000            1.00        HgI       0
+ 805.63000            2.00        HgI       0
+ 806.06000            1.00        HgI       0
+ 807.29000            5.00        HgI       0
+ 808.03000             nan        HgI       0
+ 808.26000            0.00        HgI       0
+ 809.26000             nan        HgI       0
```
Comment (wtgee) [inline pfs/lineLists/HgCd.txt:196]: @PaulPrice are the `nan`s listed in the intensity column going to cause any problems?

Comment (PaulPrice) [reply pfs/lineLists/HgCd.txt:196]: I believe they've been in there previously without any problem. But please do test this linelist on real data!

**Pull request merged**